app/services/heading_service/heading_serialization_service.rb

```diff
@@ -51,12 +48,9 @@ def serialization_service
       if heading.declarable?
         HeadingService::Serialization::DeclarableService
```

Comment:
```suggestion
        HeadingService::Serialization::NsDeclarableService
```

Reply:
Oops - this should be erroring like mad :thinking:

Reply:
Ok, realised what this is - this one is because there was originally `DeclarableService` and `NondeclarableService`, then `NsNondeclarableService` got added, and now `NondeclarableService` has been removed

I'm going to leave this as is for now I think because the `DeclarableService` is more about a 'commodities' end point for headings and I suspect all of this will get further rejigged in a future PR. If I leave it as is for now it at least makes this PR clear whats being removed rather then introducing renames alongside everything else.

But I'll do a follow up to standardise all this once the dust has settled